tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "target": "ES2015",
+    "module": "commonjs",
```

Comment:
Why are you using es2015?
It's 2021.
Please use es2020

Reply:
Changed

Reply:
Changed